src/main/scala/com.snowplowanalytics.dataflow/streaming/StreamingCounts.scala

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2017 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.dataflow.streaming
+
+/**
+ * Core of the Dataflow Streaming Application
+ * 1. Configuration information is brought in from StreamingCountsApp.scala
+ * 2. Object sets up Dataflow, PubSub and Bigtable
+ * 3. Once connections are up, Dataflow StreamingCounts stream processing starts
+ * Pub/Sub   -> Dataflow         -> Bigtable
+ * Raw Data  -> Stream Processing Data -> Stored in Database
+ */
+
+import java.lang.{Iterable => JIterable}
+
+import scala.collection.JavaConverters._
+
+import org.apache.log4j.BasicConfigurator
+
+import org.joda.time.Duration
+
+// Dataflow
+import com.google.cloud.dataflow.sdk.Pipeline
+import com.google.cloud.dataflow.sdk.io.PubsubIO
+import com.google.cloud.dataflow.sdk.options.DataflowPipelineOptions
```

Comment:
This could be `import com.google.cloud.dataflow.sdk.options.{ DataflowPipelineOptions, PipelineOptionsFactory, DataflowPipelineRunner}. Same for other imports.